ops/gulp/icons-build.js

```diff
@@ -0,0 +1,44 @@
+const gulp = require('gulp');
+const svgmin = require('gulp-svgmin');
+const svgSprite = require("gulp-svg-sprites");
+const gulpSequence = require('gulp-sequence')
+
+// svgSprite configuration
+const config = {
+    templates: { css: require("fs").readFileSync("./src/icons/icons-template.css", "utf-8") },
+    selector: "hyicon-%f",
+    common: "hyicon",
+    svg: { sprite: "css/icons.svg" },
+    cssFile: "css/icons.css",
+    preview: {sprite: "icons.html"}
+};
+
+// generate icons 
```

Comment:
This file doesn't exactly match the pattern for the other build files.

This should be the default task to build the icons where you sequence the individual tasks together — taking in the `src` and doing all the magic you need to do to output the files.  I would expect to run `gulp icons-build` to generate the icons.

Each of these tasks should be separated like`icons-generate`, `icons-copy`, etc. It may seem like overkill but it keeps this file clean as the main roadmap to understand exactly what is happening.